app/routers/user_router.py

```diff
@@ -447,7 +447,7 @@ async def list_users_randomly(current_user: str = Depends(get_current_user), db:
 
     if not actual_user["following"]:
         users = []
-        for doc in await db["users"].aggregate([{"$match": {"_id": {"$ne": actual_user["_id"]}}}, {"$sample": {"size": 100}}]).to_list(length=100):
+        for doc in await db["users"].aggregate([{"$match": {"_id": {"$ne": actual_user["_id"]}}}, {"$sample": {"size": 10}}]).to_list(length=10):
```

Comment:
This hardcoded value can come in handy with the aim of not listing all the users at the same time, in the future we can change it depending on the needs we have.